chart/templates/mayastor/obs/stats-deployment.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if .Values.stats.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-stats-aggregator
+  labels:
+    app: stats-aggregator
+    {{ include "label_prefix" . }}/release: {{ .Release.Name }}
+    {{ include "label_prefix" . }}/version: {{ .Chart.Version }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: stats-aggregator
+      {{ include "label_prefix" . }}/release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: stats-aggregator
+        {{ include "label_prefix" . }}/release: {{ .Release.Name }}
+        {{ include "label_prefix" . }}/version: {{ .Chart.Version }}
+    spec:
+      serviceAccount: {{ .Release.Name }}-service-account
+      imagePullSecrets:
+        - name: dockersecret
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{- toYaml .Values.nodeSelector | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: stats-aggregator
+          image: "{{ .Values.image.registry }}/sinhaashish/{{ .Chart.Name }}-stats-aggregator:stats"
+          command:
+            - stats-aggregator
+          ports:
+            - containerPort: 9090
+              protocol: TCP
+              name: stats
```

Comment:
A suggestion..
```suggestion
              name: stats-aggregator
```

Reply:
In one of the discussion we had , it was stated that name should just be stats @datacore-vvarakantham .